packages/native/types/base.d.ts

```diff
@@ -67,10 +67,32 @@ export type ReactNativeStyleType<Props> = Props extends {
   ? StyleType extends ReactNativeStyle ? StyleType : ReactNativeStyle
   : ReactNativeStyle
 
+export type InterpolationPrimitive<
+  StyleType extends ReactNativeStyle = ReactNativeStyle
+> =
+  | null
+  | undefined
+  | boolean
+  | number
+  | string
+  | ObjectInterpolation<StyleType>
+
 export type ObjectInterpolation<
   StyleType extends ReactNativeStyle = ReactNativeStyle
 > = StyleType
 
+export interface ArrayCSSInterpolation<
+  MergedProps,
+  StyleType extends ReactNativeStyle = ReactNativeStyle
+> extends Array<CSSInterpolation<MergedProps, StyleType>> {}
+
+export type CSSInterpolation<
```

Comment:
I don't think that `MergedProps` props are actually used anyhow inside CSSInterpolation, it only seems to be passed around, but not utilized. This is good though - with `css` you don't get access to any props at all. This means though that we can just drop this typeparam here.

Reply:
yup, absolutely. a bit of blind refactoring lands you with useless type params 😄